packages/xo-web/src/xo-app/pool/tab-network.js

```diff
@@ -81,6 +81,32 @@ class Name extends Component {
 
 // -----------------------------------------------------------------------------
 
+class AutomaticNetwork extends Component {
+  _editAutomaticNetwork = value => {
+    editNetwork(this.props.network, {
+      other_config: {
+        ...this.props.network.other_config,
+        automatic: value.toString(),
+      },
+    })
+  }
```

Comment:
With an `automatic` property, this would be as simple as:
```js
_editAutomaticNetwork = automatic => editNetwork(
  this.props.network,
  { automatic }
)
```
and all the other_config stuff would be handled in `xapi/mixins/vm.js`